### PR TITLE
Add guide for Google Cloud Engine.

### PR DESCRIPTION
**Pull request by fcuny**
This is a first attempt to document the GCE modules.

Please bear in mind that I'm not a native speaker, but I'm more than happy to improve and update this PR.

(cc @erjohnso).

Thanks.

### CODE REVIEW

**Comment by mpdehaan** [comment]
Very nice!   This comes at a very good time actually.

I hope to get this reviewed and up this week.  Thanks very much!

File: docsite/rst/guide_gce.rst [hunk 1]
```diff
@@ -0,0 +1,186 @@
+ Google Web Services Guide
```
Comment (erjohnso) [inline docsite/rst/guide_gce.rst]: Google marketing folks would prefer "Google Cloud Platform Guide" if you don't mind.

File: docsite/rst/guide_gce.rst [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+.. code-block:: bash
+
+    $ pip install apache-libcloud
+
+.. note:: If you're using Ansible on Mac OS X, libcloud needs to access a CA cert chain. You'll need to download one (you can get one for `here <http://curl.haxx.se/docs/caextract.html>`_.)
+
+Credentials
+-----------
+
+To work with the GCE modules, you'll first need some to get some credentials. You can create new one from the `console <https://console.developers.google.com/>`_ by going to the "APIs and Auth" section. Once you've created a new client ID and downloaded the generated private key (in the pkcs2 format), you'll need to convert the key by running the following command:
```
Comment (erjohnso) [inline docsite/rst/guide_gce.rst]: ...you'll first need to get some credentials.  Remove extra "some".

Comment (erjohnso) [inline docsite/rst/guide_gce.rst]: pkcs2 -> pkcs12

File: docsite/rst/guide_gce.rst [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+        local_action: gce instance_names=dev machine_type={{ machine_type }} image={{ image }} service_account_email={{ service_account_email }} pem_file={{ pem_file }} project_id={{ project_id }}
+
+secrets.py
+``````````
+
+Create a file ``secrets.py`` looking like following, and put it in some folder which is in your ``$PYTHONPATH``:
+
+.. code-block:: python
+
+    GCE_PARAMS = ('i...@project.googleusercontent.com', 'secret')
```
Comment (erjohnso) [inline docsite/rst/guide_gce.rst]: Suggest using a path notation, 'secret' -> '/path/to/project.pem'

**Comment by erjohnso** [comment]
Great work @franckcuny!  I had a few small suggestions but otherwise, this is really great to see and thank you!

**Comment by fcuny** [comment]
I've fixed all the suggestions. This should be good to go.

**Comment by mpdehaan** [comment]
Working on merging this in now!

**Comment by mpdehaan** [comment]
All merged in, I also updated this a bit to show the usage of tags rather than hostnames to manage cloud "more like cloud", and also updated somethings stylistically to make them not have to rely on "local_action" which makes formatting a little better.

Some of these changes have not had the full benefit of testing and review, so if you would like to help look these over I can easily push updates.

That all being said, this is now integrated into the docs and I'll be kicking off a docs build/push ... 
NOW.

Other upgrades are of course also welcome -- like maybe crosslinking to the GCE modules pages?

Thanks very much for this, great to see the guide sections being built out further.

Much appreciated!

**Pull request closed without merging**